src/Core/Scheduler/Providers/OGCWebServiceHelper.js

```diff
@@ -98,6 +98,22 @@ export default {
                 projection.getCoordWMTS_WGS84(tileCoord, tile.extent, tileMatrixSet);
         }
     },
+    computeTMSCoordinates(tile, extent) {
+        if (tile.extent.crs() != extent.crs()) {
+            throw new Error('Unsupported configuration. TMS is only supported when geometry has the same crs than TMS layer');
+        }
+        const c = tile.extent.center();
+        const layerDimension = extent.dimensions();
+
+        // normalize x/y
+        const x = (1 + c.x() / (layerDimension.x * 0.5)) * 0.5;
+        const y = (1 - c.y() / (layerDimension.y * 0.5)) * 0.5;
```

Comment:
does this suppose that the origin is at the center of the extent? Is it a correct assumption?